server/app/services/applicant/ReadOnlyApplicantProgramService.java

```diff
@@ -35,6 +35,12 @@ public interface ReadOnlyApplicantProgramService {
    */
   ImmutableList<Block> getAllActiveBlocks();
 
+  /**
+   * Get the {@link Block}s for this program and applicant. This includes blocks that are hidden
```

Comment:
*suggestion*: This should say it's "only" hidden blocks, not that it "includes" hidden blocks, right?